compute/gpu/troubleshooting/updating-gpu-instance-with-cloud-init.mdx

```diff
@@ -0,0 +1,34 @@
+---
+meta:
+  title: Fixing GPU issues after upgrading GPU Instances with cloud-init
+  description: This section provides information how to fix GPU issues after upgrading GPU Instances with cloud-init
+content:
+  h1: Fixing GPU issues after upgrading GPU Instances with cloud-init
+  paragraph:  This section provides information how to fix GPU issues after upgrading GPU Instances with cloud-init
+dates:
+  validation: 2022-09-19
+  posted: 2022-09-19
+---
+
+When running the `upgrade_package` command with `cloud-init` the nivida drivers may break and the GPU can become unavailable. This problem presists even after a manual reboot of the Instance and the following error displays: 
+
+```
+The GPU is not usable. The driver not loaded, with the following errors in the system journal
```

Comment:
The error message contains a grammar issue ("The driver not loaded") but if this is the actual error message as it currently displays I guess we have to leave it like this

Reply:
I don't think I can change this error message - it is coming from the Nvida drivers